spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-spi/pom.xml

```diff
@@ -6,6 +6,13 @@
 	<packaging>jar</packaging>
 	<name>spring-cloud-data-module-deployer-spi</name>
 	<description>SPI for spring-cloud-stream module deployer implementations</description>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.xd</groupId>
+			<artifactId>spring-xd-module</artifactId>
+			<version>${spring-xd-module.version}</version>
```

Comment:
you should be able to leave out the version here (and other child poms) as long as you define the whole thing (as here, including groupId, artifactId, and version) in the parent pom